src/application-manager.coffee

```diff
@@ -841,7 +841,10 @@ module.exports = class ApplicationManager extends EventEmitter
 			return @bestDeltaSource(image, available)
 		proxyvisorImages = @proxyvisor.imagesInUse(current, target)
 
-		imagesToRemove = _.filter availableAndUnused, (image) ->
+		potentialDeleteThenDownload = _.filter current.local.apps.services, (svc) ->
+			return svc.config.labels['io.balena.update.strategy'] == 'delete-then-download' and svc.status in ['Stopped']
```

Comment:
why not `svc.status == 'Stopped'` ?

Reply:
Ah, I was actually testing against `['Stopping', 'Stopped']` then I realised this was wrong, and forgot to change. Good catch!